content/docs/how-we-work/baseline/_index.md

```diff
@@ -2,12 +2,14 @@
 title: Our Reference Baseline
 weight: 1
 bookCollapseSection: true
-description: This is our reference baseline of technologies and techniques we use on a project.
+description: Adopting the baseline on a project is crucial for enhancing the overall efficiency and effectiveness of our operations.
 ---
 
 # Our Reference Baseline
 
-This is our reference baseline of technologies and techniques we use on any project. At this time, this baseline is standardized for our support projects but will eventually also apply to our managed projects.
+Adopting the baseline on a project enhances operational efficiency. It ensures quality, consistency, and clear communication for customers; reduces decision fatigue, boosts productivity, and fosters collaboration for our team; and supports scalability, compliance, and continuous improvement for Axelerant. This comprehensive approach addresses current challenges and prepares us for future growth. [Read more...](why-ref-baseline.md)
+
+At this time, this baseline is standardized for our projects in CDM but will eventually also apply to our managed projects.
```

Comment:
@Yemaneberhan-Lemma Can we link the CDM and Managed words to our pages

CDM - https://www.axelerant.com/delivery-engagement-models/continuous-development-and-maintenance
Managed - https://www.axelerant.com/delivery-engagement-models/managed-projects